testsets/crawler_google.yaml

```diff
@@ -47,3 +47,18 @@
   name: Google FeedBurner
   os: UNKNOWN
   category: crawler
+
+- target: 'Mozilla/5.0 (Linux; Android 5.0; SM-G920A) AppleWebKit (KHTML, like Gecko) Chrome Mobile Safari (compatible; AdsBot-Google-Mobile; +http://www.google.com/mobile/adsbot.html)'
+  name: AdsBot-Google-Mobile
+  os: UNKNOWN
+  category: crawler
+
+- target: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/600.2.5 (KHTML, like Gecko) Version/8.0.2 Safari/600.2.5 (Applebot/0.1; +http://www.apple.com/go/applebot)'
```

Comment:
@o-mura 
Sorry for late reply.

It contains the string `Applebot/0.1; +http://www.apple.com/go/applebot`.
Is this another Google's bot?

Reply:
I'm sorry that I missed this case. It looks an Apple's bot. Hmm.